tests/test_base.py

```diff
@@ -70,3 +70,19 @@ def test_finalize_interface(self):
         """
         visualizer = Visualizer()
         self.assertIs(visualizer.finalize(), visualizer.ax)
+
+    def test_size_property(self):
+        """
+        Test the size property on the base Visualizer
+        """
+        fig = plt.figure(figsize =(1,2))
+        visualizer = Visualizer()
+        self.assertIsNone(visualizer._size)
+        self.assertIsNotNone(visualizer.size)
+        figure_size = fig.get_size_inches() * fig.get_dpi()
+        self.assertEqual(all(visualizer.size), all(figure_size))
+        visualizer.size = (1080, 720)
+        figure_size = fig.get_size_inches() * fig.get_dpi()
+        self.assertEqual(all(visualizer.size), all(figure_size))
+        self.assertEqual(visualizer._size, (1080, 720))
+        self.assertEqual(visualizer.size, (1080, 720))
```

Comment:
@gary-mayfield last step - could we also put an assertion here that `fig.get_size_inches()` is as expected?

Reply:
From the MatPlotLib documentation fig.get_size_inches() should return a numpy array. 

Are you wanting me to do an assertion that this is the type returned before I use it?

i.e self.assertIsInstance(fig.get_size_inches(), np.ndarray)

Apologies if I have misunderstood.

Reply:
My bad, you totally did this already in line 86; I just misread it! Sorry about that!